.nvmrc

```diff
@@ -0,0 +1 @@
+v8
```

Comment:
change it to `10.14.2`. It's the same that we are using in the react-redux-base(https://github.com/rootstrap/react-redux-base/blob/master/.nvmrc)

Reply:
Actually if we run this on Node 10 we will have some problems with the tests, they behave different with that version and our current jest version (I checked on the CI). I agree to use Node 10 but we will have to upgrade jest first, our version is slightly old.